charmtools/utils.py

```diff
@@ -520,3 +520,7 @@ def prefix_lines(lines, lineno):
                    m=message)
         i += 1
     return i == 0
+
+
+class BuildError(Exception):
```

Comment:
I didn't actually mean to move this since I didn't end up using it in tactics.py like I thought I might need to.